src/css/_variables.scss

```diff
@@ -1,2 +1,7 @@
-$primary-background: radial-gradient(circle, rgba(63,94,251,1) 0%, rgba(252,70,107,1) 100%);
-;
\ No newline at end of file
+$primary-background: url("../images/mthood-bw.jpg");
+
+$transparent: rgba(0, 0, 0, 0);
+
+$opaque-white: rgba(255, 255, 255, 0.6);
```

Comment:
Quickly transitioning from HEX to rgba since they have the added opacity 'meter' there at the end. Love it.